src/guide/components/slots.md

```diff
@@ -412,6 +412,36 @@ Passing props to a named slot:
 
 Note the `name` of a slot won't be included in the props because it is reserved - so the resulting `headerProps` would be `{ message: 'hello' }`.
 
+Also note that if you are mixing named scoped slots with the default scoped slot, you need to explicitly define the default template. Otherwise, it would hint that the data of the default scoped slot would be available in the other slots scopes.
+
+```vue-html
```

Comment:
It's always a bit dangerous to have examples of incorrect code. People tend to skim over the text and just copy the examples. I think a note at the top of the example can help to mitigate that.

```suggestion
```vue-html
<!-- This template won't compile -->
```